### ISSUE TITLE
Edit content field and remove type field from avatar.yml credits file

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#2793]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, I want to edit the content field and remove the type field from a credit's yml file so that redundant code is removed and the code is easier to understand.

### Details
Currently, in each credit's yml file, there is a `content` field and a `type` field, which contains similar information. Since the fields are redundant, we decided to remove the `type` field.  

Also, we changed the `content` field to the `content-type` field to make it clearer what its purpose is. The possible values for `content-type` field are image, video, or audio; other types of content can be added if necessary. In the future, this will allow developers a way to differentiate easily between different types of content in order to show each credit's media file correctly on the website.

### Action Items
For the file `_data/internal/credits/avatar.yml`, do the following:
- [x] Change line 4 for the content field from
  ```
  content: icon
  ```
  to
  ```
  content-type: image
  ```
- [x] Remove line 11 for the type field
  ```
  type: icon
  ```
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #2775 by
  - [x] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #2775 to the New Issue Approval column and remove the Dependency label  

### Resources/Instructions
- This issue is tracked in the epic #2775.

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `_data/internal/credits/avatar.yml`

- Click on the links below to see the source file code:
https://github.com/hackforla/website/blob/0a6a6e08f231f0fd5ac2c7c9222a6bf6b8da682a/_data/internal/credits/avatar.yml#L4
https://github.com/hackforla/website/blob/0a6a6e08f231f0fd5ac2c7c9222a6bf6b8da682a/_data/internal/credits/avatar.yml#L11

**Comment by JessicaLucindaCheng** [comment]
@matt-arofin 
At the time you self-assign an issue, make sure to write in a comment the following:
- Availability: "How much time will you have this week to work on this issue?"
- ETA: "When do you expect this issue to be completed?"

**Comment by matt-arofin** [comment]
> @matt-arofin At the time you self-assign an issue, make sure to write in a comment the following:
> 
> * Availability: "How much time will you have this week to work on this issue?"
> * ETA: "When do you expect this issue to be completed?"

Ohh thank you for the reminder, I'll keep that in mind moving forward! In the meantime, here's my progress update for this week:

- `Progress`: Completed 2 action items requesting changes to the 'content' and 'type' fields of the `_data/internal/credits/avatar.yml` file.
- `Blockers`: No blockers encountered thus far.
- `Availability`: I will be blocking off 8 hours this week between Wednesday and Friday through Sunday.
- `ETA`: Currently awaiting approval and merging of my pull request, after which I hope to have this issue completed by Friday.
